src/oncall/api/v0/roster_suggest.py

```diff
@@ -0,0 +1,68 @@
+from ... import db
+from ujson import dumps as json_dumps
+from falcon import HTTPNotFound, HTTPBadRequest
+
+
+def on_get(req, resp, team, roster, role):
+    start = req.get_param_as_int('start', required=True)
+
+    connection = db.connect()
+    cursor = connection.cursor()
+
+    cursor.execute('SELECT id FROM role WHERE name = %s', role)
+    if cursor.rowcount == 0:
+        raise HTTPBadRequest('Invalid role')
```

Comment:
Won't this leak the open connection/cursor? Maybe wrap all this in a try/finally to make sure cursor and connection get closed